DataFormats/CTPPSReco/interface/CTPPSFastRecHit.h

```diff
@@ -0,0 +1,65 @@
+#ifndef CTPPSFastRecHit_H
+#define CTPPSFastRecHit_H
+
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+
+#include <vector>
+
+class CTPPSFastRecHit {
```

Comment:
the target of simulation should be to produce the same persisted objects as what's used in actual data RECO.
So, this file does not belong in  DataFormats/CTPPSReco.
If it's useful for fastsim intermediate products, FastSimDataFormats subsystem seems like the right place to have this.